### ISSUE TITLE
Code server provide a way to open a default-folder or default-workspace

### ISSUE THREAD

**Issue by zWingz** [microsoft/vscode#143580]
I hope to set a `default-folder` or `default-workspace` when the server setup.
It work on `v1.63.x` by `coder-server.sh --folder path/to/folder` but removed on https://github.com/microsoft/vscode/issues/137658

https://github.com/microsoft/vscode/issues/137658#issuecomment-1047052675

**Comment by aeschli** [comment]
@bpasero FYI
I think the request makes sense for setups like in dev containers where the server is always opened on the same folder.

**Comment by aeschli** [comment]
added new command line options `--default-folder` and `default-workspace` taking a local path, relative to the cwd.

**Comment by aeschli** [comment]
To verify start a code server with `--default-folder localFolderPath` or `--default-workspace localWorkspacePath`. 

When opening the Web UI without a `folder` argument in the browser URI the given path should be opened.
Test that the path can be relative to the current working directory or absolute.

**Comment by PhilippeVienne** [comment]
I think you just need to provide the same parameter than the URL one, it works fine on Linux side (no need of `file://` directly used the system path)

**Comment by aeschli** [comment]
Correct, @jrieken it's a local path, relative to the cwd. Something like `--default-folder ~/workspaces/foo` or `--default-folder ./foo`

**Comment by jrieken** [comment]
Well, I have tried 2 out of 3 options ;-) I think this should be handled/reported better

**Comment by joaomoreno** [comment]
While there isn't a URI error anymore, the folder gets interpreted wrongly:

![image](https://user-images.githubusercontent.com/22350/160146382-db77e8b6-3b90-4a7e-b9e2-ff5eee8abbb1.png)

But both CLI flags seem to work fine, when using FS paths.
